src/api/apiCalls.js

```diff
@@ -117,7 +117,13 @@ export const getFlashCard = async (vocabWord) => {
       throw new Error('Could not retrieve this flashcard.');
     }
     const data = await response.json();
-    return data;
+    const cleanData = {
```

Comment:
nice formatting! It's much clearer which attributes are contained in the response data